assets/styles/components/_wysiwyg.scss

```diff
@@ -0,0 +1,363 @@
+// ==========================================================================
+// Components / Wysiwyg
+// ==========================================================================
+
+.c-wysiwyg {
+    // ==========================================================================
+    // Margins
+    // ==========================================================================
+    &-first-element,
+    >:first-child {
+        margin-top: 0 !important;
+        padding-top: 0 !important;
+    }
+
+    &-last-element,
+    >:last-child {
+        margin-bottom: 0 !important;
+        padding-bottom: 0 !important;
+    }
+
+    // ==========================================================================
+    // Default
+    // ==========================================================================
+    // @include text;
+    // @include body-large;
+
+    // ==========================================================================
+    // Paragraph
+    // ==========================================================================
+    p,
+    iframe,
+    img,
+    table {
+        margin-bottom: rem(30px);
+        & + p,
+        & + ul,
+        & + ol {
+            margin-top: 1em;
+            margin-bottom: rem(30px);
+        }
+
+        & + h1,
+        & + h2,
+        & + h3,
+        & + h4,
+        & + h5,
+        & + h6 {
+            margin-top: rem(60px);
+        }
+    }
+
+    ul, ol, p {
+        line-height: 1.5;
```

Comment:
> `line-height: 1.5;`

The line height should be defined by `.c-wysiwyg` if its to be different from the root element.

Reply:
I agree, I removed this.